pkg/backup/backupinfo/manifest_handling_test.go

```diff
@@ -552,34 +548,8 @@ func TestValidateEndTimeAndTruncate(t *testing.T) {
 		},
 	} {
 		t.Run(tc.name, func(t *testing.T) {
-			// Using the expected start and end times, create a slice containing the
-			// expected expectedOrder of the manifests based on their original indexes. For
-			// example, [2, 0, 1] means that the expected output should should have
-			// the third input manifest first, the first input manifest second, and the
-			// second input manifest last.
-			expectedOrder := util.Map(tc.expected, func(ts []int) string {
-				return strconv.Itoa(slices.IndexFunc(tc.manifests, func(m backuppb.BackupManifest) bool {
-					return m.StartTime.WallTime == int64(ts[0]) && m.EndTime.WallTime == int64(ts[1])
-				}))
-			})
-			// Create URIs and locality info in a way that we can track their
```

Comment:
i think it is important to continue testing the legacy path until we delete it. it's still the default. You could keep the test as is before this diff, and below this validation, construct a `[]backupIndexMetadata` from the og `[]BackupManifest`, and rerun ValidateEndTimeAndTrucate.

Reply:
Oh yeah that's a good point. I'll do that then.